debug_rom/debug_rom.S

```diff
@@ -13,8 +13,6 @@
 
 entry:
        jal zero, _entry
-resume:
-       jal zero, _resume
```

Comment:
May I suggest again that we not change the debug entry point addresses, as it is an interface encoded inside the processor RTL?  (I do realize rocket-chip has its own debug ROM, but I am averse to this change.)

We can get the same code-size savings without changing this API by moving the FENCE from `_entry` to `entry:

    entry:
           // This fence is required because the execution may have written something
           // into the Abstract Data or Program Buffer registers.
           fence
           jal zero, _entry

    exception:
           jal zero, _exception

    _entry:

Reply:
Done. I don't understand the workflow that involves using this file for RTL, but for spike this cleanup is not really important.